CondFormats/SiPhase2TrackerObjects/BuildFile.xml

```diff
@@ -0,0 +1,11 @@
+<use name="FWCore/Framework"/>
+<use name="FWCore/Utilities"/>
+<use name="CondFormats/Serialization"/>
+<use name="DataFormats/DetId"/>
+<use name="rootrflx"/>
+<use name="boost"/>
+<use name="tbb"/>
```

Comment:
for my own education what drives this dependency:
`<use name="tbb"/>`?

Reply:
Same as previous conversation. I tested and can remove it safely.